cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractAssemblyOperator.java

```diff
@@ -98,12 +113,58 @@ protected static String name(HasMetadata resource) {
         return null;
     }
 
+    private final void reconcileCertificate(String namespace, Handler<AsyncResult<Void>> handler) {
+
+        vertx.createSharedWorkerExecutor("kubernetes-ops-pool").executeBlocking(
+                future -> {
+
+                    if (secretOperations.get(namespace, "internal-ca") == null) {
+                        log.info("Generating internal CA certificate ...");
```

Comment:
Although I personally tend to use `...` quite often, it looks bit strange in logs. Especially the second one starting with `...`.